### ISSUE TITLE
Add a function to help with recursion

### ISSUE THREAD

**Issue by dhhagan** [dhhagan/py-openaq#12]
make it easier to recursively grab all data if there is more than 1 page..

**Comment by biggydbs** [comment]
I would like to solve this issue.

**Comment by dhhagan** [comment]
@biggydbs Awesome! Let me know if you need any help or have any questions.

**Comment by biggydbs** [comment]
Yes, I tried to run all the tests, but some failed (tests.openaq_test.SetupTestCase -> test_pandasize)

**Comment by dhhagan** [comment]
That's associated with the [pandasize][1] decorator that converts the output to a Pandas DataFrame. The solution should probably be able to deal with both (or I can change the decorator).


[1]: https://github.com/dhhagan/py-openaq/blob/master/openaq/decorators.py#L29

**Comment by dhhagan** [comment]
Hey @biggydbs I just fixed some issues associated with the `pandasize` decorator; apparently some breaking changes were made to the OpenAQ API that I wasn't aware of (the API calls started failing when keyword arguments were sent that weren't allowed, whereas previously they were silently dropped). I will be merging those changes into master later today (currently are on branch dev-pc); I would love for you to merge in your changes if that was the only issue!
